### PR TITLE
Update default bootstraprc-4 properties to reflect changes in bootstrap v4 alpha 6

### LINKED ISSUE #292: Update default bootstraprc-4 properties to reflect changes in bootstrap v4 alpha 6
In the release of alpha 6 (https://github.com/twbs/bootstrap/releases/tag/v4.0.0-alpha.6) the following renames occured: 
- tag -> badge
- animation -> transitions

This is not reflected in the default properties provided with https://github.com/shakacode/bootstrap-loader/blob/master/.bootstraprc-4-default

The following lines
`animation: true`
`tags: true`

should be changed to

`transitions: true`
`badge: true`

### PR DESCRIPTION
**Pull request by mircowidmer**
- renamed 'animation' to 'transitions' and 'tags' to 'badge' (fixes #292)
- ordered default properties alphabetically
- updated example configs

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/293)

### CODE REVIEW

**Comment by apawsey** [comment]
You beat me to this :)
There are at least a couple more changes to get everything alpha6-ified...
Need to make the same changes to the examples configs, and the useFlexbox config can be dropped.

**Comment by mircowidmer** [comment]
@apawsey You are right about the dropping of useFlexbox, but I wanted to make this change only about the renaming of the two properties ;)

**Comment by justin808** [comment]
Once we have a few people verify that this PR works, I'm happy to merge and release!

**Comment by justin808** [comment]
@mircowidmer @apawsey Can both of you confirm that all the examples work.

Please also update:

CHANGELOG.md
README.md

**Comment by justin808** [comment]
@mircowidmer Any ETA?

**Comment by justin808** [comment]
@mircowidmer This looks great. Can you please

1. Confirm that you tested all the examples in /examples
2. Confirm tests and lints are passing
3. Create a changelog.md entry

---

Reviewed 1 of 1 files at r1, 3 of 3 files at r2, 3 of 3 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/293#-:-KiJsPD091TnSrvhwXui:bpsu3qf)*

**Comment by mircowidmer** [comment]
@justin808 

1. I tested the examples `basic` and `css-modules` (`npm run bs3`, `npm run bs4`)
2. I tested the example `multiple-entries` only with `npm run build`
3. `npm run test` and `npm run lint` are passing
4. In the last commit I added a changelog entry and updated the readme

One comment: `npm run bs4` in the basic example failed somewhat due to missing jQuery ("Uncaught ReferenceError: jQuery is not defined"). I could reproduce the same error before my changes: https://github.com/shakacode/bootstrap-loader/commit/3347eb376b7e1c0807f8dd4bb3c219fbdec3ae7c

I am not sure about updating the version to 3.0.0 but since it is a breaking change (e.g. the bootstrap dependency has to be updated), that seemed reasonable to me.

**Comment by justin808** [comment]
@mircowidmer I have not been updating the major version number for alpha/beta updates since it's sort of understood that those will be breaking. Also, our changes should not affect bootstrap 3 users, correct?

**Comment by mircowidmer** [comment]
So what version number are you suggesting then, I'll happily edit my last commit then. As far as I can tell, bootstrap 3 users should not be affected, yes.

**Comment by justin808** [comment]
@tc-imba -- can you confirm this one is working.
@mircowidmer -- yes no major version bump

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file. Items under
 
 Contributors: please follow the recommendations outlined at [keepachangelog.com](http://keepachangelog.com/). Please use the existing headings and styling as a guide, and add a link for the version diff at the bottom of the file. Also, please update the `Unreleased` link to compare to the latest release version.
 
-## Unreleased
+## [3.0.0] - ???
```
Comment (justin808) [inline CHANGELOG.md]: @mircowidmer this should be 2.1.0

**Comment by mircowidmer** [comment]
@justin808 I updated README and CHANGELOG and bumped version within package.json to 2.1.0.

**Pull request merged**